main.cpp

```diff
@@ -81,72 +121,131 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 } 
 
 /*
- 1)
+ 1
  */
 
+bool driveBus(int gas = 0, int stop = 2)
+{
+    ignoreUnused(gas, stop);
+    if (gas < 1)
+        return false;
+    else
+        return true;
+}
 /*
  2)
  */
-
+void delay(int time = 3, int feedback = 2)
+{
+    ignoreUnused(time, feedback);
+}
 /*
  3)
  */
+bool renderAudio(double length = 0.0, int bitDepth = 32)
+{
+    ignoreUnused(length, bitDepth);
 
+    return{};
+}
 /*
  4)
  */
+int phaser(int feedback = 0, int lfoTime = 22)
+{
+    ignoreUnused(feedback, lfoTime);
 
+    return{};
+}
 /*
  5)
  */
+int eq(int wider = 12, int gain = 2)
+{
+    ignoreUnused(wider, gain);
 
+    return{};
+}
 /*
  6)
  */
-
+void compressor(int threshold, int attack)
+{
+    ignoreUnused(threshold, attack);
+}
 /*
  7)
  */
-
+int limiter(int threshold = 12, int gain = 2)
+{
+    ignoreUnused(threshold, gain);
+    return{};
+}
 /*
  8)
  */
-
+bool clipping(int distortion = 15, int gain = 76)
+{
+    ignoreUnused(distortion, gain);
+    return{};
+}
 /*
  9)
  */
-
+bool stereo(bool mono, int wide = 15)
+{
+    ignoreUnused(wide, mono);
+    return{};
+}
 /*
  10)
  */
+bool running(bool start, bool stop)
+{
+    ignoreUnused(start, stop);
+    return{};
+}
```

Comment:
add space around your `{}`:  `return {};`
- [ ] revise, improving the readability of the code.
- [ ] `wide` is a reserved word, hence the reason it is blue here.  rename it to something else.

Reply:
add space around return {};
changed wide to spread